src/components/calculator.tsx

```diff
@@ -1,42 +1,283 @@
-import {BsFillBackspaceFill} from 'react-icons/bs'
-import {MdClear} from 'react-icons/md'
+"use client";
+
+import { BsFillBackspaceFill } from "react-icons/bs";
+import { MdClear } from "react-icons/md";
+import { TbSquareRoot2 } from "react-icons/tb";
+import { useState } from "react";
+import {
+  evaluate,
+  pi,
+  isPositive,
+  abs,
+  log,
+  factorial,
+  e,
+  sqrt,
+  sin,
+  cos,
+  tan,
+  unit,
+  ceil,
+  round,
+} from "mathjs";
+import CalcButton from "./calculatorbutton";
+
+enum CalculatorMode {
+  Standard,
+  Scientific,
+}
```

Comment:
```suggestion
const CalculatorMode = {
  Standard: "Standard",
  Scientific: "Scientific",
};
```
This could just be an object and that way you can just use it like
```ts
const [calculatorMode, setCalculatorMode] = useState<CalculatorMode>(
    CalculatorMode.Standard
  );
```
Instead of `Calculator[0]`, makes the code more readable, WDYT?

Reply:
Yeah makes sense. I was just looking for a reason to use enum 😂😂. I remember it being my favorite data type in c++ and kotlin.

Reply:
Hmmm, also the type would no longer be CalculatorMode since we are only a property from CalculatorMode and that property is a string.